app/controllers/authenticated_controller.rb

```diff
@@ -8,6 +8,18 @@ class AuthenticatedController < ApplicationController
       # Layout and view comes from GDS::SSO::ControllerMethods
       render 'authorisations/unauthorised', layout: 'unauthorised', status: :forbidden, locals: { message: e.message }
     end
+
+  else
+    before_action :require_authentication, if: :require_auth?
+    def require_auth?
+      TradeTariffAdmin.basic_session_authentication?
+    end
+
+    def require_authentication
+      unless session[:authenticated]
```

Comment:
It's probably not really an issue, but if this were to check for a specific value rather than a truthy value it would minimise the possibility of manipulating the session cookie to evade authenticating.

Reply:
My understanding is that that would be essentially impossible to achieve and if they could do that they'd just set the value to true